sympy/physics/control/lti.py

```diff
@@ -1300,6 +1300,54 @@ def to_expr(self):
             return Pow(self.den, -1, evaluate=False)
 
 
+class PIDController:
+    """
+    A PID Controller class that uses Parallel to combine the P, I, and D components.
+    Parameters
+    ==========
+    KP : Expr, Number
+        Proportional gain.
+    KI : Expr, Number
+        Integral gain.
+    KD : Expr, Number
+        Derivative gain.
+    s : Symbol
+        The complex frequency variable.
+    Examples
+    ========
+    >>> from sympy.abc import s
+    >>> from sympy.physics.control.lti import PIDController
+    >>> KP, KI, KD = 1, 0.1, 0.01
+    >>> pid = PIDController(KP, KI, KD, s)
+    >>> print(pid.to_PID_expr())
+    TransferFunction(s*(0.01*s + 1) + 0.1, s, s)
+    """
+    def __init__(self, KP, KI, KD, var):
+        if not isinstance(var, Symbol):
+            raise ValueError("var must be a Symbol")
+
+        self.KP = sympify(KP)
+        self.KI = sympify(KI)
+        self.KD = sympify(KD)
+        self.var = var
+
+        P = TransferFunction(KP, 1, var)
+        I = TransferFunction(KI, var, var)
+        D = TransferFunction(KD * var, 1, var)
+
+        self.pid_controller = Parallel(P, I, D)
+
+    def to_PID_expr(self):
```

Comment:
This has not been tested yet, kindly cover it!

Reply:
yes sir